evap/grades/tests.py

```diff
@@ -135,7 +135,7 @@ def test_upload_final_grades(self):
         evaluation.save()
         self.helper_check_final_grade_upload(course, 0)
 
-    def test_toggle_no_grades(self):
```

Comment:
Let's also check that sending the same request again doesn't change anything by adding another request at the end of the test

Reply:
Done, please close